index.js

```diff
@@ -49,6 +55,23 @@ function Clay(config, customFn, options) {
         console.log(JSON.stringify(error));
       });
     });
+  } else if (typeof Pebble !== 'undefined') {
```

Comment:
We don't do anything. If there is no Pebble global Clay is in a development environment.